tests/misc/tgenconstraints.nim

```diff
@@ -2,7 +2,7 @@ discard """
   errormsg: "cannot instantiate T2"
   file: "tgenconstraints.nim"
   line: 25
-  disabled: true
+    disabled: true
```

Comment:
Created disabled around 2014-04 in a commit that references #1075
		Running gives the output:
			Expected:
			cannot instantiate T2

			Gotten:
			undeclared identifier: 'TNumber'

Reply:
This test doesn't make sense, there isn't a TNumber, expectation seems wrong.

Reply:
I'll take this as a delete.